OMGVA PoS/ClientApp/src/pages/UpdateBusinessPage.tsx

```diff
@@ -0,0 +1,68 @@
+﻿import React, { useState, useEffect } from 'react';
+import { useParams, useNavigate, useLocation } from 'react-router-dom'
+import { Business, updateBusiness } from '../services/businessService';
+import { getTokenBusinessId, getTokenRole } from '../utils/tokenUtils';
+import '../index.css';
+import BusinessDataForm from '../components/Form/BusinessDataform';
```

Comment:
This import is weird. While building with npm, this line returns an error. Maybe we could rename this? For example, we could have a folder called "BusinessForm" instead of "Form" and then within that folder we could rename the file to index.tsx? (There is a folder called "LoginForm", so having "LoginForm" and "Form" at the same level would be a bit confusing. Additionally, this way, we would follow some kind of pattern)

Reply:
It turns out that this error is for building, which we will not use. There is a point in improving consistency of folder structure though.

Reply:
Also I think I fixed it while moving folders.